packages/slice-machine/src/components/TextLink/TextLink.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC, ReactNode } from "react";
+import { isValidElement, cloneElement } from "react";
+
+import clsx from "clsx";
+import { Text } from "@prismicio/editor-ui";
+
+import * as styles from "./TextLink.css";
+import * as textStyles from "@prismicio/editor-ui/dist/components/Text/Text.css";
+
+type IconProps = Readonly<{
+  className?: string;
+}>;
+
+type Variant = keyof typeof textStyles.variant;
+
+export interface TextLinkProps {
+  children: string;
+  color?: keyof typeof styles.colorVariant;
```

Comment:
We are adding the color but here only support primary, right?

Reply:
Yes, it's just a way to extend it. Editor uses indigo instead of purple. Indigo would have a "secondary" key, but we need to discuss it with design team